cookbook-react-native/recipes/hooks/debounce/useDebounce.ts

```diff
@@ -0,0 +1,33 @@
+import { useEffect, useState } from 'react';
+import { Nullable } from '@interfaces/globalInterfaces';
+
+const DEFAULT_DELAY = 500;
```

Comment:
this is half a second, i would increase the default delay

Reply:
Do you think 750 would be more suitable?

Reply:
i would say a whole second

Reply:
Done!